/**:
  ros__parameters:
    queue_size: 5

    realsense_devices: ["realsense_front", "realsense_ground"]

    realsense_color_info_topic: "color/camera_info"
    realsense_color_topic: "color/image_raw"
    realsense_depth_info_topic: "depth/camera_info"
    realsense_depth_topic: "depth/image_raw"

    realsense_adaptor_ouput_topic: "synced_image"